Issue importing `pykeops_nvrtc`

Hi,

I had some trouble with the line 
https://github.com/getkeops/keops/blob/dcec73cb066ecf4c13efe7612a585022123d6224/pykeops/pykeops/common/keops_io/LoadKeOps_nvrtc.py#L20

The `build_folder` is added to `sys.path` when pykeops is imported. But if the `build_folder` is manually removed from `sys.path` later, the `pykeops_nvrtc` will not be find. As an example

```python
import sys
from copy import deepcopy
syspath = deepcopy(sys.path)

import pykeops

# Reset sys.path to its value before importing pykeops
sys.path = syspath

pykeops.test_torch_bindings()
``` 

leads to a import error. Seems that this modification
```python
pykeops_nvrtc = importlib.import_module( get_build_folder() + "/pykeops_nvrtc")
```
can make this part of the code robust to malicious `sys.path` modifications.

@Louis-Pujol , interresting. 

Not sure it is related, but : I have also notice that under a new python venv. If keops is imported undirectly (`from pykeops.torch import Vi` say) keopscore (in fact ld) fails to detect libnvrt...

```
[KeOps] Compiling cuda jit compiler engine ... 
[KeOps] Warning : There were warnings or errors :
/usr/bin/ld: cannot find -lnvrtc: No such file or directory
collect2: error: ld returned 1 exit status
```
Importing directly works:

```
$ python -m pykeops
 
[KeOps] Compiling cuda jit compiler engine ... OK
[pyKeOps] Compiling nvrtc binder for python ... OK

```

--- a/pykeops/pykeops/common/keops_io/LoadKeOps_nvrtc.py
+++ b/pykeops/pykeops/common/keops_io/LoadKeOps_nvrtc.py
@@ -1,4 +1,5 @@
 import os
+import sys
 
 import keopscore.config.config
 from keopscore.config.config import get_build_folder
@@ -16,6 +17,10 @@ class LoadKeOps_nvrtc_class(LoadKeOps):
 
     def init_phase2(self):
         import importlib
+        if pykeops.get_build_folder() not in sys.path:
+            # The build folder is supposed to be in the python path, if not,
+            # we add it
+            sys.path.append(pykeops.get_build_folder())
 
         pykeops_nvrtc = importlib.import_module("pykeops_nvrtc")
 

